slackhealthbot/database/models.py

```diff
@@ -56,14 +56,16 @@ class FitbitUser(TimestampMixin, AsyncAttrs, Base):
     last_sleep_end_time: Mapped[Optional[datetime]] = mapped_column()
     last_sleep_sleep_minutes: Mapped[Optional[int]] = mapped_column()
     last_sleep_wake_minutes: Mapped[Optional[int]] = mapped_column()
-    latest_activities: Mapped[list["FitbitLatestActivity"]] = relationship(
-        back_populates="fitbit_user", lazy="select", cascade="all, delete-orphan"
+    activities: Mapped[list["FitbitActivity"]] = relationship(
+        back_populates="fitbit_user",
+        lazy="select",
+        cascade="all, delete-orphan",
+        order_by="desc(FitbitActivity.updated_at)",
```

Comment:
Is this the place to put an `order_by` clause? 🤔

Reply:
Removed this default order by, and added a crud function to get the latest activity for a user and activity type.